internal/manifest/status/condition.go

```diff
@@ -74,3 +74,14 @@ func ConfirmResourcesCondition(manifest *v1beta2.Manifest) {
 		manifest.SetStatus(status.WithOperation(resourceCondition.Message))
 	}
 }
+
+func ConfirmInstallationCondition(manifest *v1beta2.Manifest) {
```

Comment:
quick and easy unit tests can be written for this func

Reply:
done